compiler/damlc/tests/daml-test-files/ProposalIou.EXPECTED.rst

```diff
@@ -33,10 +33,14 @@ Templates
   + **Choice Burn**
     
 
-Template Instances
-^^^^^^^^^^^^^^^^^^
+Data Types
+^^^^^^^^^^
```

Comment:
Same here @associahedron - what is the correct daml-doc for instances of generic templates?

Reply:
Same as above. Without the docs marker there is no way for damldocs to detect that a type is a template instance.